src/feature/common/Sidebar/SidebarNode.tsx

```diff
@@ -86,12 +89,19 @@ const SidebarNode = ({
   autogeneratedCategory,
   icon,
 }: SidebarNodeProps) => {
-  if (node.name === "ROOT") {
+  debugger;
+  if (isRoot(node)) {
     return (
       <div>
         {node.children.map((c) => {
           // Actually needs to be called as a function to achieve depth-first traversal,
           // otherwise React lazily evaluates it and doesn't recurse immediately, resulting in breadth-first traversal.
+          let category, autogeneratedCategory, icon;
```

Comment:
Maybe add some explicit types for these? (`string | undefined` etc.) Editor gives warnings about implicit `any` types.